hs_core/tasks.py

```diff
@@ -184,6 +184,9 @@ def manage_task_nightly():
         # send email for people monitoring and follow-up as needed
         send_mail(subject, email_msg, settings.DEFAULT_FROM_EMAIL, [settings.DEFAULT_SUPPORT_EMAIL])
 
+
+@periodic_task(ignore_result=True, run_every=crontab(minute=0, hour=0, day_of_week="monday"))

```

Comment:
@aphelionz Most likely putting ```monday``` string for day_of_week would work, but from the documentation I don't see an example confirming ```monday``` is an accepted string, just to be safe, setting it to be integer ```1``` (representing Monday) is probably better. Also, it is probably better to have this task to run at a slightly different time than the DOI task, perhaps setting minute to 15 rather than 0.